docs/json.md

```diff
@@ -0,0 +1,50 @@
+# JSON over TChannel
+
+This document outlines how we encode JSON over TChannel.
+
+For JSON call requests the `as` (arg scheme) transport header
+must be set to `json`. Requests will be made with `call req`
+messages and responses will be sent using `call res` messages,
+with values for `arg1`, `arg2` and `arg3` as defined in
+[Arguments][].
+
+For each `call req`, the service name (`service~1`) should be
```

Comment:
`service~1` is a reference to the ~1 syntax of the binary protocol. i.e. 1byte length